docs/tutorials/trainers.ipynb

```diff
@@ -47,7 +47,8 @@
    },
    "outputs": [],
    "source": [
-    "%pip install torchgeo"
+    "%pip install torchgeo\n",
+    "%pip install tensorboard"
```

Comment:
```suggestion
    "%pip install torchgeo tensorboard"
```
If you don't install them at the same time, the second installation may require changing versions of the former.